org.gnome.Games.LibretroPlugin.PicoDrive.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "org.gnome.Games.LibretroPlugin.PicoDrive",
+    "runtime": "org.gnome.Games",
+    "runtime-version": "stable",
+    "sdk": "org.gnome.Sdk//3.34",
+    "build-extension": true,
+    "appstream-compose": false,
+    "finish-args": [
+        /* Needed to find the ROMs */
+        "--filesystem=home:ro"
+    ],
+    "cleanup": ["/include", "/lib/pkgconfig",
+                "/share/pkgconfig", "/share/aclocal",
+                "/man", "/share/man", "/share/gtk-doc",
+                "*.la", "*.a",
+                "/lib/girepository-1.0",
+                "/share/dbus-1", "/share/doc", "/share/gir-1.0"
+    ],
```

Comment:
Nitpick: I'm sure you don't need most of these

Reply:
Right. None of them, in fact.